code/modules/goonchat/browserassets/js/browserOutput.js

```diff
@@ -723,8 +723,8 @@ $(function() {
 		$("body").css('line-height', savedConfig.slineHeight);
 		internalOutput('<span class="internal boldnshit">Loaded line height setting of: '+savedConfig.slineHeight+'</span>', 'internal');
 	}
-	if(savedConfig.sdarkmode == 'false'){
```

Comment:
I mean just `if(saved.Config.sdarkmode == opts.darkmode){`

Reply:
That doesn't work. I have no clue why, but it doesn't. Doing that results in it always opening white if it's `==` and dark if it's `!=`

Reply:
Oh, one of these is string the other is bool.